tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/templates/AbstractLiferayComponentTemplate.java

```diff
@@ -96,7 +96,7 @@
 
     private final static String[][] DEPENDENCY=
     {
-        { "com.liferay.portal", "com.liferay.portal.kernel", "2.0.0"},
+        { "com.liferay.portal", "com.liferay.portal.kernel", "2.6.0"},
```

Comment:
because the quickfix will add missed dependecy while user added new component and the targetPlatform always return the minium version of portal kernel is 2.6.0. So we set this to 2.0.0 that is no any useful.